packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -783,11 +784,13 @@ export class ApolloServerBase {
   }
 
   private ensurePluginInstantiation(plugins?: PluginDefinition[]): void {
-    if (!plugins || !plugins.length) {
-      return;
+    const pluginsToInit = [...plugins || []];
```

Comment:
Would a default argument `[]` have the desired effect?

Reply:
It would!  However, in reality, this code changes in an upcoming PR, so I'm going to leave it as is here — as valid as a suggestion as this is. 😄

Reply:
To make this more clear: I ended up incorporating this feedback in a preparatory way in the branch that this PR merges into (https://github.com/apollographql/apollo-server/pull/3988/commits/4b59b02b0da2602ea1f0bcb399687ad170699e07) rather than trying to bring it back later.  Good move, I think.  Thanks for noting this.